### ISSUE TITLE
can use in urp?

### ISSUE THREAD

**Issue by zero16832** [keijiro/TestbedHDRP#19]
can use in urp?

**Comment by keijiro** [comment]
Sorry, I can't understand what you meant (maybe it's too short and simple to understand). Could you rephrase it?

**Comment by zero16832** [comment]
this shader can use in urp mode?

**Comment by keijiro** [comment]
What is "this"? Which shader in this project (there are lots of shaders).

**Comment by zero16832** [comment]
Voxelizer.shader thx your help

**Comment by zero16832** [comment]
i just want to know how to make this project to run on urp mode? it can do that?how to?

**Comment by keijiro** [comment]
> Voxelizer.shader thx your help

It requires a significant amount of modification to port it to URP. I also don't recommend it as noted on README.

**Comment by keijiro** [comment]
> i just want to know how to make this project to run on urp mode? it can do that?how to?

This project is a testbed for HDRP. Obviously, it doesn't support URP as there is no compatibility between HDRP and URP.

**Comment by zero16832** [comment]
thanks
